### PR TITLE
Fix long chirps being cut off scren

### PR DESCRIPTION
**Pull request by dayan9265**
This PR fixes the problem with long chirp with no spaces and no special characters (such as / f.ex) to not be completely shown onscreen. It also fixes the problem with the trashcan not being gray when offline. Here is how it looks like now:

image

### CODE REVIEW

**Comment by Tyratox** [comment]
As mentioned yesterday I think it would be good to move the polyfill to `core/platform`, rename it to something more general & import it in App.tsx. Otherwise LGTM 😄

**Review by MeKHell — APPROVED** [review]
LGTM

File: fe1-web/src/features/social/components/ChirpCard.tsx [hunk 1]
```diff
@@ -264,6 +265,7 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
                     name="delete"
                     testID="delete_chirp"
                     onPress={() => setShowDeleteConfirmation(true)}
+                    disabled={reactionsDisabled.delete}
                     size="small"
```
Comment (Tyratox) [inline fe1-web/src/features/social/components/ChirpCard.tsx:269]: Good catch! 😁

**Review by Tyratox — APPROVED** [review]
LGTM 😁

**Pull request merged**